src/_posts/hands-on-ii.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Hands-On! Simple Physical Behaviours In A-Frame
+author: twitter|salvadelapuente|Salvador de la Puente
+date: 2017-08-08 12:01:00
+layout: blog
+image:
+  src: hands-on-ii.png
+---
+
+This article is the second in a series showing how to build a hand-based, interactive training experience in VR for a circuit manipulation procedure. In the [first part](/blog/hands-on-i/), I chose a [specific step in the procedure](https://www.youtube.com/embed/tHP2kX6aAZM?rel=0&amp;showinfo=0) and prototyped the workspace, tools and materials used in that step with simple geometrical primitives. This time around I will add basic physics to the elements of the scene and you will learn about A-Frame’s entity-component-system architecture.
+
+<!--more-->
+
+I’ve implemented this series using a VIVE. Remember that for hand-based interactions, **a cardboard or Daydream is not enough**. You will need an advanced head-mounted display supporting room-scale VR and [6DoF](https://en.wikipedia.org/wiki/Six_degrees_of_freedom) controllers such as those included in the [HTC VIVE](https://www.vive.com/us/product/#controller-intro) or the [Oculus Touch](https://www.oculus.com/rift).
```

Comment:
If we collapse the three articles into one we can remove these redundant parts. Since we're publishing them at the same time it's something I would consider. We're in the short attention span age :)